src/test/java/seedu/address/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -1,71 +1,68 @@
-package seedu.address.logic.commands;
```

Comment:
This might be nitpicky, but wouldn't it nice to still have tests for the descriptor inner class?